CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 This changelog will be used to generate documentation on [release notes page](http://azure.microsoft.com/en-us/documentation/articles/app-insights-release-notes-dotnet/).
 
 ## Version 2.5.0-beta2
-
+- Remove calculation of sampling-score based on Context.User.Id
```

Comment:
Can you add link to the issue as well? So that it's easier for someone reading change log to understand more about the change.
https://github.com/Microsoft/ApplicationInsights-dotnet/issues/625

Reply:
Done.